### DISCUSSION TITLE
Mixed use of async-std and tokio?

### DISCUSSION THREAD

**Discussion by shanesveller** [hendrikmaus/kube-leader-election#6] [Q&A] [answered]
Hello! I noticed that the crate's dev dependencies and code examples use `async-std`, but `kube` and its dependencies already depend upon Tokio. Is it a conscious/intentional decision to mix the two async runtimes within one application? What was the motive for that?

**Comment by hendrikmaus** [comment]
I am glad you asked. It was an unconscious decision as my journey with `async` in Rust began with `async-std` and I haven't had any issue in projects with both `tokio` and `async-std` in them. However, I would be _really keen_ to know if there are implications and how significant they are.

For this crate, the next logical step is, of course, to provide `tokio` based examples.

**Comment by hendrikmaus** [comment, marked as answer]
Resolved in https://github.com/hendrikmaus/kube-leader-election/pull/57
